package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "mozilla-leadership-network",
+  "name": "network",
```

Comment:
Yeah, that's the name of what would be the package if we published this project to npm. I'm making that consistent with the repo name per convention.